parser/parser.go

```diff
@@ -1579,8 +1578,16 @@ func (p *parser) parseCallOrConversion(fun ast.Expr, isCmd bool) *ast.CallExpr {
 	p.exprLev++
 	var list []ast.Expr
 	var ellipsis token.Pos
+	var extractTuple bool
 	for p.tok != endTok && p.tok != token.EOF && !ellipsis.IsValid() {
-		list = append(list, p.parseRHSOrType()) // builtins may expect a type: make(some type, ...)
+		expr := p.parseRHSOrType(isCmd && len(list) == 0)
+		if tuple, ok := expr.(*tupleExpr); ok {
+			list = tuple.Items
+			extractTuple = true
```

Comment:
maybe we don't need `extractTuple`, but change `isCmd = false`

Reply:
fixed